### PR TITLE
[FCE-400] Support names in demo app (and add missing CI checks)

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

Allow test app to handle user names in similar way as mobile app

Also add missing lint/format CI checks. And reformat code using our rules

## Motivation and Context

Easier app testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: examples/react-client/videoroom/src/RoomConnector.tsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ function psersistValues({ roomManagerUrl, roomName, userName }: FormProps) {
 function getPersistedValues() {
   return {
     defaultRoomManagerUrl: localStorage.getItem("roomManagerUrl") ?? "",
-    defaultRoomName: localStorage.getItem("userName") ?? "",
-    defaultUserName: localStorage.getItem("roomName") ?? "",
+    defaultRoomName: localStorage.getItem("roomName") ?? "",
+    defaultUserName: localStorage.getItem("userName") ?? "",
```
Comment (mironiasty) [inline examples/react-client/videoroom/src/RoomConnector.tsx:20]: bug fix 🙈

File: examples/react-client/videoroom/src/App.tsx [hunk 2]
```diff
@@ -67,7 +67,9 @@ function App() {
                 )}
 
                 <div className="absolute bottom-2 left-0 w-full grid place-content-center text-center text-xs z-30">
-                  <p className="bg-slate-100/60 px-1 rounded-sm">{id}</p>
+                  <p className="bg-slate-100/60 px-1 rounded-sm">
+                    {(metadata as { name?: string })?.name ?? id}
```
Comment (mironiasty) [inline examples/react-client/videoroom/src/App.tsx:71]: main change

File: examples/react-client/videoroom/package.json [hunk 3]
```diff
@@ -7,6 +7,9 @@
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint:check": "eslint . --ext .ts,.tsx",
+    "format": "prettier --write . --ignore-path ./.eslintignore",
+    "format:check": "prettier --check . --ignore-path ./.eslintignore",
     "preview": "vite preview"
```
Comment (mironiasty) [inline examples/react-client/videoroom/package.json:13]: Missing commands

File: examples/react-client/videoroom/src/App.tsx (hunk 2 above)
Comment (czerwiukk) [reply examples/react-client/videoroom/src/App.tsx:71]: that inline type is not the prettiest, but will do the job until we have the predefined metadata typed by the sdk

**Review by czerwiukk — APPROVED** [review]

File: examples/react-client/videoroom/src/App.tsx (hunk 2 above)
Comment (mironiasty) [reply examples/react-client/videoroom/src/App.tsx:71]: That is my assumption - soon we should have these values typed better, so then it will be fixed.

**Pull request merged**